servers/republik/modules/crowdfundings/graphql/schema-types.js

```diff
@@ -63,13 +67,27 @@ type PackageOption {
   createdAt: DateTime!
   updatedAt: DateTime!
 
+  # for pledgeOptions
   amount: Int
-  templateId: ID
+
+  # for custom packages
+  customization: PackageOptionCustomization
```

Comment:
I'd vote for flattening: `membership` and `membershipPeriods` directly on the option.